.ci/continuous.release.cloudbuild.yaml

```diff
@@ -59,13 +72,17 @@ steps:
       - "install-dependencies"
     env:
       - 'GOPATH=/gopath'
+      - 'CGO_ENABLED=1'
+      - 'GOOS=darwin'
+      - 'GOARCH=arm64'
+      - 'CC=aarch64-linux-gnu-gcc'
```

Comment:
We need to update `name: golang:1` to ubuntu in order to have the right system packages. We should also add a tag to the ubuntu container to pin the version as a best practice